diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2c48fab9..b2f20832 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -273,8 +273,8 @@ OPTION(PICOQUIC_FETCH_PTLS "Fetch PicoTLS during configuration" OFF)
 if(PICOQUIC_FETCH_PTLS)
     include(FetchContent)
     FetchContent_Declare(picotls
-        GIT_REPOSITORY      https://github.com/h2o/picotls.git
-        GIT_TAG             f350eab60742138ac62b42ee444adf04c7898b0d)
+        GIT_REPOSITORY      https://github.com/afrind/picotls.git
+        GIT_TAG             314a1397a26f290c7ada57813d33dcdff1e859e8)
     FetchContent_MakeAvailable(picotls)
 endif()
 
